en_partut-ud-train.conllu

```diff
@@ -30623,7 +30623,7 @@
 27	in	in	ADP	E	_	28	case	_	_
 28	1960	1960	NUM	N	NumType=Card	20	obl	_	_
 29	to	to	ADP	E	_	32	case	_	_
-30	more	more	ADV	B	Degree=Cmp	32	advmod	_	_
+30	more	more	ADV	B	Degree=Cmp|ExtPos=ADV	32	advmod	_	_
```

Comment:
upos=ADJ

Reply:
Can do.  So you would agree that `less than #` is equally `fixed` like `more than`?

Reply:
With quantities they are both `fixed`: https://universaldependencies.org/en/dep/fixed.html#approximators-quantity-modifiers